example/properties.yml

```diff
@@ -0,0 +1,12 @@
+---
+dbs:
+  address: 10.244.1.2
+  databases:
+    - name: autoscaler
+      tag: autoscaler
```

Comment:
Tag can be used to find a database object from database list using p_arr. But as this is not being used in our case so removed.